src/Magnum/Audio/Test/SourceTest.cpp

```diff
@@ -46,6 +71,102 @@ void SourceTest::debugState() {
     CORRADE_COMPARE(out.str(), "Audio::Source::State::Playing\n");
 }
 
+void SourceTest::position() {
+    Source source{};
+    constexpr Vector3 pos{3, 5, 6};
+    source.setPosition(pos);
+
+    CORRADE_COMPARE(source.position(), pos);
+}
+
+void SourceTest::direction() {
+    Source source{};
+    constexpr Vector3 dir{3, 1, 2};
+    source.setDirection(dir);
+
+    CORRADE_COMPARE(source.direction(), dir);
+}
+
+void SourceTest::velocity() {
+    Source source{};
+    constexpr Vector3 vel{-3, 5, -6};
+    source.setVelocity(vel);
+
+    CORRADE_COMPARE(source.velocity(), vel);
+}
+
+void SourceTest::gain() {
+    Source source{};
+    constexpr Float gain = 0.1234;
+    source.setGain(gain);
+
+    CORRADE_COMPARE(source.gain(), gain);
+}
+
+void SourceTest::looping() {
+    Source source{};
+    source.setLooping(true);
+    CORRADE_COMPARE(true, source.isLooping());
+    source.setLooping(false);
+    CORRADE_COMPARE(false, source.isLooping());
+}
+
+void SourceTest::relative() {
+    Source source{};
+    source.setRelative(true);
+    CORRADE_COMPARE(true, source.isRelative());
```

Comment:
```cpp
source.setRelative(true);
CORRADE_VERIFY(source.isRelative());
source.setRelative(false);
CORRADE_VERIFY(!source.isRelative());
```

In case of boolean values it's shorter to write :)